src/api/entity/ideaComment.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  CreateDateColumn,
+  UpdateDateColumn,
+  DeleteDateColumn,
+  Column,
+  ManyToOne,
+} from 'typeorm';
+import {User, Idea} from '.';
+
+@Entity('idea_comments')
+export class IdeaComment {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @CreateDateColumn({type: 'timestamp', default: () => 'NOW()'})
+  createdAt: Date;
+
+  @UpdateDateColumn({type: 'timestamp', default: () => 'NOW()'})
+  updatedAt: Date;
+
+  @DeleteDateColumn({type: 'timestamp', nullable: true})
+  deletedAt: Date;
+
+  @Column({type: 'text'})
+  comment: string;
+
+  @ManyToOne(() => Idea, (idea: Idea) => idea.likes)
```

Comment:
ini bukannya `(idea) => idea.comments` ya?

Reply:
@yofriadi di-resolve dulu ya.. bener harusnya kan idea.commnets no like in idea commnets, like cuma ada di idea.

Reply:
oiya okee, nice catch thankyou